README.rst

```diff
@@ -262,6 +262,21 @@ To enable LTI-DL and its capabilities, you need to set these settings in the blo
 
 To enable LTI-NRPS, you set **Enable LTI NRPS** to **True** in the block settings on Studio.
 
+
+LTI 1.1/1.2 Basic Outcomes Service 1.1
+--------------------------------------
+
+This XBlock supports `LTI 1.1/1.2 Basic Outcomes Service 1.0 <http://www.imsglobal.org/spec/lti-bo/v1p1/>`_. Please see these
+`instructions <https://github.com/openedx/xblock-lti-consumer/tree/master/docs/basic_outcomes_service.rst>`_
```

Comment:
I've never been able to get document cross-referencing to work in Github RST files, so I've hardcoded the instructions locations. Let me know if you know how to cross-reference documents!